COMETwebapp/Components/ParameterEditor/ParameterTable.razor.cs

```diff
@@ -89,5 +89,33 @@ protected override void OnInitialized()
 
             this.closeEditor = new EventCallbackFactory().Create(this, () => { this.ViewModel.IsOnEditMode = false; });
         }
+
+        /// <summary>
+        /// Customizes the table rows
+        /// </summary>
+        /// <param name="e">The <see cref="GridCustomizeElementEventArgs"/></param>
+        private void OnCustomizeElement(GridCustomizeElementEventArgs e)
+        {
+            if (e.ElementType == GridElementType.DataRow)
+            {
+                var isPublishable = (bool)e.Grid.GetRowValue(e.VisibleIndex, nameof(ParameterBaseRowViewModel.IsPublishable));

```

Comment:
Is it not possible to directly get the ParameterBaseRowViewModel instance ? and then grab the IsPublishable value?

Reply:
Yes, just updated the PR with this change